jemalloc/jemalloc.go

```diff
@@ -1,5 +1,6 @@
 package jemalloc
 
+// Does it mean that Jemalloc will be used as default allocator in Rust?
```

Comment:
It is will be used thruout the entire binary. So, yes, both Rust and Go will use jemalloc.